ext/pdo_pgsql/tests/pdopgsql_setNoticeCallback.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+PdoPgsql::setNoticeCallback()
+--EXTENSIONS--
+pdo
+pdo_pgsql
+--SKIPIF--
+<?php
+require __DIR__ . '/config.inc';
+require dirname(__DIR__, 2) . '/pdo/tests/pdo_test.inc';
+PDOTest::skip();
+?>
+--FILE--
+<?php
+
+require_once __DIR__ . "/config.inc";
+
+$db = new PdoPgsql($config['ENV']['PDOTEST_DSN']);
+
+function disp($message) { echo trim($message)."\n"; }
+function attach($db) { $db->setNoticeCallback('disp'); }
+
```

Comment:
nit: would it be possible to add a case where the callback is invalid ?

Reply:
Here it is, with `$rounds` that concisely lists the tried "callbacks".

Reply:
Now that both endpoints were grouped, we have 3 cases of bad callbacks:
- an int in pdopgsql_setNoticeCallback.phpt
- a non-resolvable string in pdopgsql_setNoticeCallback.phpt
- a trampoline that throws a BadMethodCallException at runtime in issue78621_method.phpt